packages/web-app-admin-settings/src/components/Users/ContextActions.vue

```diff
@@ -26,11 +37,31 @@ export default defineComponent({
     const instance = getCurrentInstance().proxy as any
 
     const filterParams = computed(() => ({ resources: props.items }))
+    const selectedPersonalDrives = reactive([])
+    watch(
```

Comment:
in case someone wonders, no, computed prop wont work here because it wont be reactive